### ISSUE TITLE
Bump Snowflake JDBC

### ISSUE THREAD

**Issue by chuwy** [snowplow-archive/snowplow-snowflake-loader#2]
Snowflake JDBC 3.2.1 was available at the time on 0.1.0 release, but assembly failed on Travis with odd error: 

```
   - [error] java.lang.RuntimeException: deduplicate: different file contents found in the following:
   - [error] /home/travis/.ivy2/cache/org.apache.httpcomponents/httpclient/jars/httpclient-4.5.2.jar:mozilla/public-suffix-list.txt
   - [error] /home/travis/.ivy2/cache/net.snowflake/snowflake-jdbc/jars/snowflake-jdbc-3.2.1.jar:mozilla/public-suffix-list.txt
   - [error] 	at sbtassembly.Assembly$.applyStrategies(Assembly.scala:141)
   - [error] 	at sbtassembly.Assembly$.x$1$lzycompute$1(Assembly.scala:25)
   - [error] 	at sbtassembly.Assembly$.x$1$1(Assembly.scala:23)
   - [error] 	at sbtassembly.Assembly$.stratMapping$lzycompute$1(Assembly.scala:23)
   - [error] 	at sbtassembly.Assembly$.stratMapping$1(Assembly.scala:23)
   - [error] 	at sbtassembly.Assembly$.inputs$lzycompute$1(Assembly.scala:67)
   - [error] 	at sbtassembly.Assembly$.inputs$1(Assembly.scala:57)
   - [error] 	at sbtassembly.Assembly$.apply(Assembly.scala:84)
   - [error] 	at sbtassembly.Assembly$.$anonfun$assemblyTask$1(Assembly.scala:249)
   - [error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:44)
   - [error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:42)
   - [error] 	at sbt.std.Transform$$anon$4.work(System.scala:64)
   - [error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:257)
   - [error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
   - [error] 	at sbt.Execute.work(Execute.scala:266)
   - [error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:257)
   - [error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:167)
   - [error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:32)
   - [error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
   - [error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
   - [error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
   - [error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
   - [error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
   - [error] 	at java.lang.Thread.run(Thread.java:748)
   - [error] (loader/*:assembly) deduplicate: different file contents found in the following:
   - [error] /home/travis/.ivy2/cache/org.apache.httpcomponents/httpclient/jars/httpclient-4.5.2.jar:mozilla/public-suffix-list.txt
   - [error] /home/travis/.ivy2/cache/net.snowflake/snowflake-jdbc/jars/snowflake-jdbc-3.2.1.jar:mozilla/public-suffix-list.txt
   - [error] Total time: 17 s, completed Aug 31, 2017 1:25:51 PM
```

**Comment by chace86** [comment]
What about this PR resolved the issue? I am using snowflake jdbc 3.12.7

**Comment by chuwy** [comment]
Hi @chace86. Could you please post your question on the support forums: discourse.snowplowanalytics.com/
